code/modules/mob/living/carbon/slime/items.dm

```diff
@@ -270,7 +270,10 @@
 	G.set_species(golem_type)
 	G.name = G.species.get_random_name()
 	G.real_name = G.name
-	to_chat(G, SPAN_NOTICE("You are a golem. Serve your master, and assist them in completing their goals at any cost."))
+	G.accent = G.species.default_accent
+	G.preEquipOutfit(/datum/outfit/admin/golem, FALSE)
+	G.equipOutfit(/datum/outfit/admin/golem, FALSE)
```

Comment:
Not in favor of handing out a id card that can copy access to non-antags.

Reply:
likewise, that's a bit too much, a better solution would be golems getting abilities to force doors open or such